helpdesk-bot/src/test/java/org/symphonyoss/symphony/bots/helpdesk/bot/api/V1HelpDeskControllerTest.java

```diff
@@ -89,6 +91,9 @@
   @Mock
   private HelpDeskAi helpDeskAi;
 
+  @Mock
+  private MessagesClient messagesClient;
```

Comment:
I think it's not required. Please remove this field and its corresponding import.